dev/io.openliberty.microprofile.lra.1.0.internal/resources/io/openliberty/microprofile/lra/internal/resources/Lra.nlsprops

```diff
@@ -0,0 +1,27 @@
+###############################################################################
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+
+
+#ISMESSAGEFILE true
+#NLS_ENCODING=UNICODE
+#COMPONENTPREFIX CWMRX
+
+# LRA is currently using the message prefix CWMRX in the range 5000-5999
+# This is temporary and should be fixed with a proper prefix before GA.
+
+LRA_INVALID_COORDINATOR_URI.CWMRX5000E=CWMRX5000E: The LRA coordinator URI is not valid
+LRA_INVALID_COORDINATOR_URI.CWMRX5000E.explanation=LRA participants must be able to contact the coordinator.
+LRA_INVALID_COORDINATOR_URI.CWMRX5000E.useraction=Correct the LRA coordinator configuration in the server.xml
```

Comment:
Add "file." to the end of the useraction so that it has "Correct the LRA coordinator configuration in the server.xml file."